extensions/wikia/AdEngine/js/video/player/ui/toggleFullscreen.js

```diff
@@ -0,0 +1,29 @@
+/*global define*/
+define('ext.wikia.adEngine.video.player.ui.toggleFullscreen', [
+	'wikia.document',
+	'wikia.log'
+], function (doc, log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.video.player.ui.toggleFullscreen';
+
+	function add(video) {
+		var toggleFullscreenButton = doc.createElement('div');
+		toggleFullscreenButton.classList.add('toggle-fullscreen');
+
+		// TODO: replace with proper image asset in CSS
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[project-id]: wikia:app